README.md

```diff
@@ -1,2 +1,77 @@
+
+FreeKiss changes the layout of the bookmarks page to include the mangas' image cover.
+
+![Comparison between the default display and FreeKiss display](/../Screenshots/Images/Screenshots/Enhanced_Display.png)
+*On the left is the default display of the bookmarks while the right side is the display while FreeKiss "Enhanced display" option is enabled.*
+
+### Bookmarks manager
+> Context: Didn't like being forced to go to the bookmarks page to manage my bookmarks. So now it's available at other places on the website.
+
+The bookmarks managers allow you to manage your bookmarks (add/remove, change status *(see next section for status)*, mark as read/unread) on other pages than the bookmarks page.
+
+The managers can be enabled on the frontpage, the mangas pages and the chapters pages.
+
+![Example of the managers on the frontpage](/../Screenshots/Images/Screenshots/Manager_Example.png)
+
+*Here is an example of the manager for some of the mangas on the frontpage.*
+
+### Bookmarks status
+> Context: As an ex-user of MangaFox, the feature that I missed the most on KissManga was the possibility to sort bookmarks with a status, such as "On Hold" or "Plan to read". So here you go.
+
+FreeKiss adds a status option to your bookmarks. If you enable the option, a new status button will appear on the bookmark managers. Using this button, it is possible to assign a status to any bookmarks.
+
+Once the option is enabled, the bookmarks will also be sorted on your bookmarks page.
+
+![Sorted bookmarks page](/../Screenshots/Images/Screenshots/Sort_Status.png)
+*It is possible to add the status without the enhanced display. Here is a screenshot of the two possible displays.*
+
+The currently available status are "Default", "On Hold" and "Plan to read".
+
+![Available status](/../Screenshots/Images/Screenshots/Status.png)
+
+### Advanced options
+
+Using the extension popup, it is possible to access the Advanced Options which allow you to enable or disable most of FreeKiss' features.
+
+![Advanced Options](/../Screenshots/Images/Screenshots/Advanced_Options.png)
+
+### Export bookmarks
+
+While accessing the Advanced Options you will find a tab allowing you to export your bookmarks.
+
+Three formats are currently available: plain text, XML and JSON.
+
+![Advanced Options](/../Screenshots/Images/Screenshots/Export.png)
+
+The current output is not really nice, so if you use this feature and have a suggestion do not hesitate to open an issue.
+
+## Coding style
+
+Before you grab some torches to come burn my house, I want to say that I am no javascript expert as it is not the language I am the most fluent with (don't really use it aside from writing this extension), so it is possible that part of the code is ugly. However, I am a pretty chill and open minded guy, so if you find something that you consider as an abomination that should normally be avoided at all cost, just send me a message and I'll look into it.
```

Comment:
You apparently merged master into develop and brought the README with it. I'd prefer if the README was not on develop, as I directly edit it on master and the one on develop would slowly be outdated. Can you revert the README ? (or alternatively replace the content with the old one)